quickwit-storage/src/bundle_storage.rs

```diff
@@ -132,8 +138,11 @@ impl Storage for BundleStorage {
         Err(unsupported_operation(path))
     }
 
-    async fn copy_to_file(&self, path: &Path, _output_path: &Path) -> crate::StorageResult<()> {
-        Err(unsupported_operation(path))
+    async fn copy_to_file(&self, path: &Path, output_path: &Path) -> crate::StorageResult<()> {
+        let file_content = self.get_all(path).await?;
```

Comment:
This loads the entire file in anonymous memory. 
That's often 10+ GB. 

Can you fix that (Note it is non trivial with the current storage API) or at least document that?

Reply:
I changed it load chunks of 100MB. For a better solution with the storage API, we would need something like `append_from_slice`